build-system/compile/bundles.config.js

```diff
@@ -1,5 +1,8 @@
 'use strict';
+const bentoBundles = require('./bundles.config.bento.json');
 const extensionBundles = require('./bundles.config.extensions.json');
+
+exports.bentoBundles = bentoBundles;
```

Comment:
dupe of line 213?

Reply:
Jinx!

Reply:
Good catch! Not sure how that happened, maybe a rebase issue?